IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_ElectricFlowStorageDeviceTypeRecharger/DocPropertySet.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_ElectricFlowStorageDeviceTypeRecharger" UniqueId="deaeec80-79ed-1711-8c77-6113891ee76e" ApplicableType="IfcElectricFlowStorageDevice/RECHARGER" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
+	<Properties>
+		<DocProperty xsi:nil="true" href="NominalPower_2pu16hiH1CqQv04BNgxAph" />
+		<DocProperty xsi:nil="true" href="NominalCurrent_0sDhk0qUWHuO00025QrEV" />
+		<DocProperty xsi:nil="true" href="NominalFrequency_0rcYS0qU0HuO00025QrEV" />
```

Comment:
What do these relate to compared to the rated values on PSet_ElectricalDeviceCommon? please clarify

Reply:
Agree with you! Has been erased. 840d7b293d3051eca8050639884528e00dfc5d4e